.github/workflows/dispatch-k6-performance.yml

```
@@ -56,6 +56,13 @@ on:
           - 'tests/k6/tests/serviceowner/performance/serviceOwnerRandomSearch.js'
           - 'tests/k6/tests/enduser/performance/enduserRandomSearch.js'
           - 'tests/k6/tests/graphql/performance/graphqlRandomSearch.js'
+          - 'tests/k6/tests/graphql/performance/graphqlGetAllDialogsForEnduser.js'
+          - 'tests/k6/tests/graphql/performance/graphqlGetAllDialogsForParties.js'
+          - 'tests/k6/tests/graphql/performance/graphqlGetAllDialogsForEnduserFTS.js'
+          - 'tests/k6/tests/graphql/performance/graphqlGetAllDialogsForPartiesFTS.js'
+          - 'tests/k6/tests/graphql/performance/graphqlGetAllDialogsForParties.js'
```

actionlint (1.7.7)

option "tests/k6/tests/graphql/performance/graphqlGetAllDialogsForParties.js" is duplicated in options of "testsuitepath" input

(events)